docs/Makefile

```diff
@@ -64,6 +64,7 @@ html: templates cli
 	$(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
 	@echo
 	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
+	@cp ../docker/compose/sawtooth-demo.yaml $(BUILDDIR)/html/app_developers_guide/sawtooth-demo.yaml
```

Comment:
In the documentation below, it uses 'sawtooth-default.yaml' not 'sawtooth-demo.yaml', so the last argument here should probably be '$(BUILDDIR)/html/app_developers_guide/sawtooth-default.yaml'